test/setup.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs');
+var path = require('path');
+var walk = require('walk');
+var util = require('./util');
+
+var specDir = path.join(__dirname, 'spec');
+
+// deal with CRLF from windows folks writing javascript :(
+function readCleanCRLF(fpath) {
+  return fs.readFileSync(fpath, 'utf8')
+    .trim()
+    .replace(/\r\n/gm, '\n');
+}
+
+// TODO: Replace me.
+function getFolderName(filePath) {
+  // windows
+  if (filePath.indexOf('/') == -1) {
+    return filePath.substring(filePath.lastIndexOf('\\') + 1, filePath.length);
+  }
+
+  // unix
+  return filePath.substring(filePath.lastIndexOf('/') + 1, filePath.length);
+}
+
+// Returns a promise to a map of records containing
+// { specFileDir: { missingExpectFile: bool, missingSourceFile: bool }}
+function getSpecFiles(specPath) {
+  return new Promise(function (resolve,) {
```

Comment:
should it raise an error if it has the `skip` file since it doesn't do anything now?